templates/status.html

```diff
@@ -14,63 +14,85 @@
 {% endblock breadcrumbs %}
 
 {% block content %}
-  <h1 class="text-3xl break-words pt-2 pb-4 md:pt-0 md:pb-6 md:text-4xl font-bold text-slate-900">
-    Status
-  </h1>
-  <div class="grid gap-8 md:grid-cols-2 lg:grid-cols-3 mb-8">
-    {% for backend in backends %}
-      {% #card %}
-        <div class="px-4 py-2 sm:px-6 sm:py-3">
-          <div class="flex flex-wrap items-center justify-between sm:flex-nowrap">
-            <h2 id="workspaces-title" class="text-xl font-semibold tracking-tight text-slate-900">
-              {{ backend.name }}
-            </h2>
-            {% if backend.show_warning %}
-              {% pill variant="warning" text="Missing" %}
-            {% endif %}
+  <div class="prose">
+    <h1>OpenSAFELY Status</h1>
+    <p>This page provides information about the status of the OpenSAFELY platform.</p>
+    <p>OpenSAFELY consists of {% link href="https://docs.opensafely.org/technical-architecture/" text="several interconnected systems, websites, and databases" new_tab="true" %} that work together to provide a secure, transparent, open-source software platform for analysis of electronic health records data.</p>
+
+    <h2>Systems status</h2>
+    <p>The real-time status of the indivdual components that make up the OpenSAFELY platform can be viewed on the {% link href="https://status.opensafely.org/" text="OpenSAFELY status website &nearr;&#xFE0E;" append_after="." new_tab="true" %}</p>
+
+    <h2>Backend status</h2>
+    <p>A 'backend' is a secure, live data environment, managed by a provider of electronic health records data.</p>
+    <p>The status of each backend which can run jobs for OpenSAFELY researchers can be viewed below.</p>
+
+    <h3>TPP</h3>
+    {% db_maintenance_banner %}
+
+    <div class="grid gap-8 mb-8">
+      {% for backend in backends %}
+        {% #card %}
+          <div class="px-4 py-2 sm:px-6 sm:py-3">
+            <div class="flex flex-wrap items-center justify-between sm:flex-nowrap">
+              <h2 id="workspaces-title" class="text-xl font-semibold tracking-tight text-slate-900">
+                {{ backend.name }}
+              </h2>
+              {% if backend.show_warning %}
+                {% pill variant="warning" text="Missing" %}
```

Comment:
Have you checked what this looks like when you have multiple backends and only one of them has a warning banner enabled?